### PR TITLE
Add profiles plots to all TC tests, make them appear first in artifacts

### PR DESCRIPTION
**Pull request by trontrytel**
This PR adds the plot with final profiles of TC single column simulations to the buildkite artifacts of all TC single column test cases. (Before it was only done for Bomex). It also changes the final plot title so that it should appear first in the list of artifacts (easier to find among all the generated mp4 animations).

I think we need this for the explicit/implicit edmf work

### CODE REVIEW

**Review by charleskawczynski — APPROVED** [review]

**Comment by charleskawczynski** [comment]
Hmm, maybe we forgot to reset the mse tables in another PR?

File: examples/hybrid/define_tc_quicklook_profiles.jl [hunk 1]
```diff
@@ -4,7 +4,7 @@ using Plots
 function plot_tc_profiles(folder, hdf5_filename)
 
     input_filename = joinpath(folder, hdf5_filename)
-    output_filename = joinpath(folder, "final_profiles.png")
+    output_filename = joinpath(folder, "____________________________________final_profiles.png")
```
Comment (charleskawczynski) [inline examples/hybrid/define_tc_quicklook_profiles.jl:7]: 🙀

Comment (trontrytel) [reply examples/hybrid/define_tc_quicklook_profiles.jl:7]: lol, I have no idea how else to make the profiles plots stick out from all the animations

**Comment by trontrytel** [comment]
Added some case twice to buildkite. Will rebase one more time

File: examples/hybrid/define_tc_quicklook_profiles.jl (hunk 1 above)
Comment (charleskawczynski) [reply examples/hybrid/define_tc_quicklook_profiles.jl:7]: Fair point

Comment (trontrytel) [reply examples/hybrid/define_tc_quicklook_profiles.jl:7]: The formatter also does not like me 😹

**Comment by trontrytel** [comment]
Hmm... @charleskawczynski I got another mse not regressed error here and I just rebased. Seems like there is something not deterministic going on? I restarted the simulation in question, but I'm pretty sure that this PR should not change anything

**Pull request merged**